Segment-MoEngage.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "Segment-MoEngage"
-  s.version          = "7.5.0"
+  s.version          = "8.0.0"
```

Comment:
This should be a separate commit in the master branch. should not be changed here.